notebooks/cshore/cshore_example.ipynb

```diff
@@ -217,7 +198,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "cshoreio = wrr.cshoreio(pathPrefix='/Users/rdchlth9/Codes/cmtb_refactor/notetbooks/cshore/data', fNameBase=fname+'_cmtb', versionPrefix='base',\n",
+    "cshoreio = wrr.cshoreio(workingDirectory='/Users/rdchlth9/Codes/cmtb_refactor/notetbooks/cshore/data', testName=fname+'_cmtb', versionPrefix='base',\n",
```

Comment:
@thesser1  can we make these paths generic (relative or something like [/your/path/here])